server/py/services/api/initial_data.py

```diff
@@ -993,6 +1002,46 @@ def _create_project_summaries(db, db_session):
     db._upsert(db_session, project_summaries, ignore=True)
 
 
+def _init_system_id(db_session: sqlalchemy.orm.Session):
+    # Initializes a system id for MLRun deployment.
+    # The system id is first checked in the database. If it does not exist, the function checks for a configured id
+    # (from environment variables or config), and if neither is found, a new random one is generated and stored.
+    db = framework.db.sqldb.db.SQLDB()
+
+    # check if a system id already exists in the database
+    system_id = db.get_system_id(db_session)
+
+    if system_id is not None:
+        logger.info("Existing system id found in the database", system_id=system_id)
+        return
+
+    # check if the system id is configured via environment variables or configmap
```

Comment:
the configmap is just a way to set mlrun config via env vars.
You actually only check if the config is already set.